splinterd/src/config/env.rs

```diff
@@ -16,6 +16,7 @@ use std::env;
 
 use crate::config::{ConfigError, ConfigSource, PartialConfig, PartialConfigBuilder};
 
+const CONFIG_DIR: &str = "CONFIG_DIR";
```

Comment:
Following the convention of the other two environment variables - the name of the const should be CONFIG_DIR_ENV and the environment variable should be SPLINTER_CONFIG_DIR.